src/TLuaInterpreter.h

```diff
@@ -399,6 +396,14 @@ class TLuaInterpreter : public QThread
     static int getProfileName( lua_State * );
     static int raiseGlobalEvent( lua_State * );
 
+public slots:
+    void slot_replyFinished( QNetworkReply * );
+    void slotPurge();
+    void slotDeleteSender();
+
+private:
+    TGatekeeperThread* mpGatekeeperThread;
```

Comment:
Unless you have just created something to use this, `mpGatekeeperThread` is __cruft__ and can be removed.